### ISSUE TITLE
[JENKINS-71211] Experimental flag value display name cut off on 13" screen

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/jenkins#18636]
<p>been there since implementation: <a href="https://github.com/jenkinsci/jenkins/pull/7299" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/7299</a></p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/60378/60378_image-2023-05-06-10-58-43-932.png" style="border: 0px solid black" /></span></p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/timja.png" title="timja's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=timja">timja</a>, imported from: <a class="no-jira-link-rewrite" href="https://issues.jenkins.io/browse/JENKINS-71211" target="_blank">Experimental flag value display name cut off on 13" screen</a></i></summary>
<i><ul>
<li><b>status</b>: Open
<li><b>priority</b>: Minor
<li><b>component(s)</b>: core
<li><b>label(s)</b>: ux
<li><b>resolution</b>: Unresolved
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 2025-11-24
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>been there since implementation: <a href="https://github.com/jenkinsci/jenkins/pull/7299" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/jenkinsci/jenkins/pull/7299</a></p>

<p><span class="image-wrap" style=""><img src="https://issues.jenkins.io/secure/attachment/60378/60378_image-2023-05-06-10-58-43-932.png" style="border: 0px solid black" /></span></p></pre>
</details>
</details>
<ul><li><i>environment</i>: <code>2.404-SNAPSHOT - <a href="https://github.com/jenkinsci/jenkins/pull/7948">https://github.com/jenkinsci/jenkins/pull/7948</a></code></li></ul>
<details><summary><i>1 attachment</i></summary>

- [image-2023-05-06-10-58-43-932.png](https://issues.jenkins.io/secure/attachment/60378/image-2023-05-06-10-58-43-932.png)
  > ![image-2023-05-06-10-58-43-932.png](https://issues.jenkins.io/secure/attachment/60378/image-2023-05-06-10-58-43-932.png)
</details>

**Comment by AbdelHamdyGhanem** [comment]
This has been resolved. Prototype.js was fully removed from Jenkins core starting with the 2.426 weekly release (October 2023) and confirmed removed from the November 2023 LTS line. The migration required updates in about 60 plugins.
Reference: https://www.jenkins.io/blog/2023/10/09/prototype-removed/
Since core no longer bundles Prototype and the associated remove-prototype.flag experimental flag has since been removed as well (nothing left to toggle), I think this can be closed.

<img width="1902" height="858" alt="Image" src="https://github.com/user-attachments/assets/3a33f228-5ed7-4277-87f8-bd8d4bb8a13a" />

**Comment by timja** [comment]
I think a better layout would work.

Whilst there's nothing cut off currently its borderline, easy to edit the HTML to make it break though:

<img width="1021" height="329" alt="Image" src="https://github.com/user-attachments/assets/831cbcdb-fd5c-4581-9962-7cf51a0fa0e8" />

This is what it looks like with no edits which isn't great:

<img width="991" height="68" alt="Image" src="https://github.com/user-attachments/assets/497709d4-d8af-4500-a0e8-dbaa3401f5e8" />

**Comment by AbdelHamdyGhanem** [comment]
> I think a better layout would work.
> 
> Whilst there's nothing cut off currently its borderline, easy to edit the HTML to make it break though:
> 
> <img alt="Image" width="1021" height="329" src="https://private-user-images.githubusercontent.com/21194782/618912796-831cbcdb-fd5c-4581-9962-7cf51a0fa0e8.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODM1MjcyOTIsIm5iZiI6MTc4MzUyNjk5MiwicGF0aCI6Ii8yMTE5NDc4Mi82MTg5MTI3OTYtODMxY2JjZGItZmQ1Yy00NTgxLTk5NjItN2NmNTFhMGZhMGU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNzA4VDE2MDk1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJkNGNkNWQyOWJhNzhlNzhkMjMxNTAyOWJiNGVhYjUyNGEyNDI1MWNlNmFjMjY3ZTU0MTkyZDM4NTZmYWU0MTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.G9fx3eXd1T7QWhRcld6TvsalsGM9ts2LNuAfwqV5eDM">
> This is what it looks like with no edits which isn't great:
> 
> <img alt="Image" width="991" height="68" src="https://private-user-images.githubusercontent.com/21194782/618913159-497709d4-d8af-4500-a0e8-dbaa3401f5e8.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3ODM1MjcyOTIsIm5iZiI6MTc4MzUyNjk5MiwicGF0aCI6Ii8yMTE5NDc4Mi82MTg5MTMxNTktNDk3NzA5ZDQtZDhhZi00NTAwLWEwZTgtZGJhYTM0MDFmNWU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNzA4VDE2MDk1MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRlMGNiODBmYjIzZjhhNTM1ZDgxNTQ4NzQ0YWRkNGNlOWYyOTdjYWEyM2RmMDdjMzYzZWZiYTBkZTQ5YTMyNjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.H6ZE-39icXy1pGtMkvbDso8EfQtD0R9mXToFmZhb0yk">

You're right, reproduced it. At exactly 1280px width (13" laptop resolution), the Value column dropdown clips "Default (disabled)" down to "Default (disable" with the text just cut off, no ellipsis. Screenshot attached. running 2.571-SNAPSHOT.
Agree the real fix is a better layout for this table (e.g. the Value column needs a min-width, or the dropdown should truncate with text-overflow: ellipsis instead of hard-clipping, or the table should shift to a stacked/responsive layout below a certain width) rather than relying on the current content staying just short enough to fit.

<img width="1086" height="1002" alt="Image" src="https://github.com/user-attachments/assets/e72187c3-f871-49ff-a0c4-191fa19b22b5" />

**Comment by AbdelHamdyGhanem** [comment]
Created a PR for this. Please let me know of this is what you are looking for:
https://github.com/jenkinsci/jenkins/pull/27047
